docs/development/setup-vagrant.md

```diff
@@ -94,112 +94,38 @@ Now you are ready for [Step 2: Get Zulip Code.](#step-2-get-zulip-code).
 
 #### Ubuntu
 
-If you're in a hurry, you can copy and paste the following into your terminal
-after which you can jump to [Step 2: Get Zulip Code](#step-2-get-zulip-code):
-
-```
-sudo apt-get -y purge vagrant && \
-wget https://releases.hashicorp.com/vagrant/2.2.4/vagrant_2.2.4_x86_64.deb && \
-sudo dpkg -i vagrant*.deb && \
-sudo apt-get -y install build-essential git ruby lxc lxc-templates cgroup-lite redir && \
-vagrant plugin install vagrant-lxc && \
-vagrant lxc sudoers
-```
-
-For a step-by-step explanation, read on.
-
-##### 1. Install Vagrant
-
-For Ubuntu 18.04 Bionic or newer, you can just install from `apt`:
-
-```
-sudo apt install vagrant
-```
-
-For Ubuntu 16.04 Xenial and 14.04 Trusty, you'll need a more recent
-version of Vagrant than what's available in the official Ubuntu
-repositories.
-
-First uninstall any vagrant package you may have installed from the Ubuntu
-repository:
-
-```
-christie@ubuntu-desktop:~
-$ sudo apt-get purge vagrant
-```
-
-Now download and install the .deb package for [Vagrant][vagrant-dl].  E.g.:
+##### 1. Install Vagrant, Docker, and Git
 
 ```
 christie@ubuntu-desktop:~
-$ wget https://releases.hashicorp.com/vagrant/2.2.4/vagrant_2.2.4_x86_64.deb
-
-christie@ubuntu-desktop:~
-$ sudo dpkg -i vagrant*.deb
+$ sudo apt install vagrant docker.io git
 ```
 
-##### 2. Install remaining dependencies
-
-Now install git and lxc-related packages:
+##### 2. Add yourself to the `docker` group:
 
 ```
 christie@ubuntu-desktop:~
-$ sudo apt-get install build-essential git ruby lxc lxc-templates cgroup-lite redir
+$ sudo adduser $USER docker
+Adding user `christie' to group `docker' ...
+Adding user christie to group docker
+Done.
 ```
 
-##### 3. Install the vagrant lxc plugin:
-
-```
-christie@ubuntu-desktop:~
-$ vagrant plugin install vagrant-lxc
-Installing the 'vagrant-lxc' plugin. This can take a few minutes...
-Installed the plugin 'vagrant-lxc (1.2.1)'!
-```
-
-If you encounter an error when trying to install the vagrant-lxc plugin, [see
-this](#nomethoderror).
-
-##### 4. Configure sudo to be passwordless
-
-Finally, [configure sudo to be passwordless when using Vagrant LXC][avoiding-sudo]:
+You will need to reboot for this change to take effect.  If it worked,
```

Comment:
Let's add a parenthetical that you don't need to reboot if your user was already in the `docker` group.